.github/workflows/sonarcloud.yml

```
@@ -41,27 +41,24 @@ permissions:
 jobs:
   Analysis:
     runs-on: ubuntu-latest
+    if: secrets.SONAR_TOKEN != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)